src/components/Class/ClassList/index.js

```diff
@@ -11,12 +11,24 @@ function ClassList() {
 
   const { loading, data = [] } = useSelector(({ Class }) => Class.allClasses);
 
-  useEffect(() => {
-    dispatch(classActions.getClasses());
-  }, [dispatch]);
+  useEffect(
+    (e) => {
```

Comment:
@komalahire can you remove 'e'. Also, did you not get any eslint errors when trying to make a git commit?

Reply:
Done,  yes I got error I removed also but i forget do again git add.

Reply:
Done